play/PlayerRules/Basic Rules.txt

```diff
@@ -236,8 +330,28 @@ Luck:
 ╠═══════╬═══════════════════════════════════════════════════════════════════════════════════════╣
 ║ 9-10  ║ Nothing bad happens... this time							║
 ╚═══════╩═══════════════════════════════════════════════════════════════════════════════════════╝
+
+Nanites:
+	Although not a primary stat, The maximum number of Nanites you can use at a given time is determined
+	by your character class and an associated primary stat. Your class may tell you its "Class Primary Attribute" 
+	is Strength. In that case, Strength * 10 is your maximum Nanites. More generally:
+	    Your Maximum Nanites = (Your Class's Primary Attribute) * 10

```

Comment:
Yes, after a discussion with Taylor we both thought it was more prudent to use our existing resource pool (nanites) instead of adding a secondary one and splitting abilities between the two pools. Reasoning: less complex. Although in retrospect, having Nanites separated out as something that only activates abilities that are super-human would be a way to devalue nanites. Which might mean that using a class primary attribute system for stamina or nanites might be more feasible.

Or.. that gives me an idea. What if we attached a stamina resource/attribute to FORT points, and made all the athletic active feats use stamina instead of Nanites? That would significantly improve FORT's value.

Reply:
1. The extreme example is medic, who we probably don't want to have to manage health, stamina, nanites, adrenaline, *and* contagion resources.
2. We would still run into all of the design problems we've talked about with tying health and a resource to the same stat.
3. Also we would be indirectly hurting characters who never take athletic feats as part of their aesthetic.
So, I am currently in favor of having one general ability-focused resource pool.

Reply:
Good points, fair enough.